src/main/java/com/zextras/transcoders/primitive/AbstractBinaryValueTranscoder.java

```diff
@@ -0,0 +1,22 @@
+package com.zextras.transcoders.primitive;
+
+import com.zextras.utils.TranscoderUtils;
+
+/**
+ * Helper class for decoding a string to binary data.
+ */
+public abstract class AbstractBinaryValueTranscoder<T> implements ValueTranscoder<T> {
+
+  public AbstractBinaryValueTranscoder() {
```

Comment:
Use protected modifier here instead of public. (public constructors should not be used in abstract classes, as constructors of abstract classes can only be called in constructors of their subclasses)
please fix other occurrences too.

Reply:
done!